sqrl-planner/sqrl-common/src/main/java/org/apache/calcite/jdbc/SqrlSchema.java

```diff
@@ -107,6 +109,9 @@ public void addRelationship(Relationship relationship) {
     this.sysTableToRelationshipMap.put(relationship.getFromTable(), relationship);
     plus().add(String.join(".", relationship.getPath().toStringList()) + "$"
         + sqrlFramework.getUniqueMacroInt().incrementAndGet(), relationship);
+    if (!relationship.getParameters().isEmpty()) {
+      plus().add(String.join(".", relationship.getPath().toStringList()), relationship);
```

Comment:
This should call a helper function. We should already have one for this.

Reply:
Removed in next commit